_pages/guides/data-hub-framework.markdown

```diff
@@ -54,21 +147,3 @@ There are some downsides to this approach:
 
 6. Run `./gradlew mlDeploy` from inside the `marklogic` directory. This command will deploy the new configuration, but it will not change the DHF project's content database, because you have removed all related configuration files.
```

Comment:
The build.gradle likely needs to be edited, because setSchemaPermissions should be disabled (or removed altogether). DHF will be responsible of applying the correct doc permissions to the schemas (and triggers)

Reply:
Steps are added to remove those parts.